app/wizard.py

```diff
@@ -1,79 +1,8 @@
-from os import getenv
-from pathlib import Path
-import sys
-
-from PyQt6.QtWidgets import (
-    QApplication,
-    QSizePolicy,
-    QWizard,
-)
-
-from app.acme_directory_configuration_parser import ACMEDirectoryConfigurationParser
-from app.pkcs_lib_finder import PKCS11LibFinder
-
-from .pkcs import pkcs
-from .appacme import ACME
-from .page.welcome import WelcomePage
-from .page.selectkey import SelectYubiKeyPage
-from .page.logindigid import LoginWithDigiDPage
-from .page.creatersakey import CreateRSAKeysPage
-from .page.requestcert import RequestCertificatePage
-from .page.savetoyubi import SaveToYubiKeyPage
-
-import urllib.parse
+from app.main import ApplicationBootstrapper
 from dotenv import load_dotenv
 
-PROJECT_ROOT = Path(__file__).parent.parent
-
-DEFAULT_ACME_CA_SERVER_URL = "https://acme.proeftuin.uzi-online.rdobeheer.nl/directory"
-DEFAULT_YUBIKEY_PIN = "123456"
-DEFAULT_PROEFTUIN_OIDC_LOGIN_URL = "https://proeftuin.uzi-online.irealisatie.nl"
-
-
-class MainWindow(QWizard):
-    def __init__(self, mypkcs, myacme, oidc_provider_base_url: urllib.parse.ParseResult):
-        super().__init__()
-
-        self.setWindowTitle("YubiKey Wizard")
-        self.resize(1024, 768)
-
-        self.setSizePolicy(QSizePolicy.Policy.Expanding, QSizePolicy.Policy.Expanding)
-        self.addPage(WelcomePage())
-        self.addPage(SelectYubiKeyPage(mypkcs))
-        self.addPage(CreateRSAKeysPage(mypkcs))
-        self.addPage(LoginWithDigiDPage(myacme, oidc_provider_base_url))
-        self.addPage(RequestCertificatePage(mypkcs, myacme))
-        self.addPage(SaveToYubiKeyPage(mypkcs))
-
-        # When the wizard has finished, close the application
-        self.finished.connect(QApplication.instance().quit)
-
 
 if __name__ == "__main__":
     load_dotenv()
```

Comment:
Is it not weird that the "main" logic is now in the main file but we should start the application through `python -m app.wizard` instead of `python -m app.main`? Now we have the main file I think the wizard file is not needed.

Further the `load_dotenv` is executed twice.